src/Traits/HasTries.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Saloon\Traits;
+
+use Saloon\Contracts\Request;
+use Saloon\Exceptions\Request\RequestException;
+use Saloon\Exceptions\Request\FatalRequestException;
+
+trait HasTries
+{
+    /**
+     * The number of tries a request will be attempted
+     *
+     * Set to null to disable the retry functionality.
+     */
+    public ?int $tries = null;
+
+    /**
+     * The interval between attempting to retry a request
+     *
+     * Set to null to disable the interval.
+     */
+    public ?int $retryInterval = null;
```

Comment:
This is only a personal preference, but I always find it really helpful when the property name or doc block says the unit that the interval/time should be in (seconds, milliseconds, microseconds, etc). I guess it's not too hard to dive into the code and figure it out, but it's nice to see it at a quick glance because I'm lazy haha!

Reply:
Good point! I've given the docblocks a new lick of paint 🙌

![Screenshot 2023-09-05 at 19 41 55](https://github.com/saloonphp/saloon/assets/29132017/5d7c09b0-4600-41f6-b7a7-c001b31315d2)